checklist.md

```diff
@@ -23,7 +23,7 @@ public class Main {
 
 #### Write informative messages in methods.
 Use english only and make them informative:
-message should indicate what type of `Machine` is working right now `Truck`, `Bulldozer` or `Excavator`.
+message should indicate what type of `core.basesyntax.Machine` is working right now `core.basesyntax.Truck`, `core.basesyntax.Bulldozer` or `core.basesyntax.Excavator`.
```

Comment:
Why did you change the checklist file?

Reply:
It`' because i tryed to create new classes in other package. But when i did it MainAdd didn't see this classes. So i decided to refactor this classes to package "core.basesyntax". And when i did it "core.basesyntax" was added everywhere before name of classes.